packages/klighd-core/src/skgraph-model-renderer.ts

```diff
@@ -77,4 +78,12 @@ export class SKGraphModelRenderer extends ModelRenderer {
             })
             .filter(vnode => vnode !== undefined) as VNode[]
     }
+
+    /** Renders a node as a proxy. */
+    renderProxy(node: SKNode): VNode | undefined {
+        this.forceRendering = true;
+        const vnode = super.renderElement(node);
+        this.forceRendering = false;
```

Comment:
What if something else set the forceRendering to true and it should not be reset? This method could just restore the old state of forceRendering instead of setting it to false.

Reply:
Good remark, I didn't think about others potentially using the field I added aswell